test/unit/image_data_test.rb

```diff
@@ -30,6 +28,18 @@ class ImageDataTest < ActiveSupport::TestCase
     assert image_data.valid?
   end
 
+  test "requires images larger than 960x640 to be cropped" do
+    image_data = build_example("960x960_jpeg.jpg")
+    assert_not image_data.valid?
+    assert_equal ["must be 960px wide and 640px tall, but is 960px wide and 960px tall"], image_data.errors[:file]
```

Comment:
I think updating the `:needs_cropping` error message I mentioned will help make this test more specific, making it a more specific cropping error message rather than a regular wrong size error. Is there also a way of checking that we get an error with type `:needs_cropping` in this test? Using similar syntax from the `edition_images_controller` maybe something like `assert image_data.errors.of_kind?(:file, :needs_cropping)`?